sagemaker-python-sdk/tensorflow_script_mode_quickstart/tensorflow_script_mode_quickstart.ipynb

```diff
@@ -0,0 +1,274 @@
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!git clone https://github.com/sherjilozair/char-rnn-tensorflow"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This repository includes a [README.md](https://github.com/sherjilozair/char-rnn-tensorflow/blob/master/README.md#basic-usage) with an overview of the project, requirements, and basic usage:\n",
+    "\n",
+    "> #### **Basic Usage**\n",
+    "> _To train with default parameters on the tinyshakespeare corpus, run **python train.py**. \n",
+    "To access all the parameters use **python train.py --help.**_\n",
+    "\n",
+    "[train.py](https://github.com/sherjilozair/char-rnn-tensorflow/blob/master/train.py#L11) uses the Python [argparse](https://docs.python.org/3/library/argparse.html) library and requires the following arguments:\n",
+    "\n",
+    "```python\n",
+    "parser = argparse.ArgumentParser(formatter_class=argparse.ArgumentDefaultsHelpFormatter)\n",
+    "# Data and model checkpoints directories\n",
+    "parser.add_argument('--data_dir', type=str, default='data/tinyshakespeare', help='data directory containing input.txt with training examples')\n",
+    "parser.add_argument('--save_dir', type=str, default='save', help='directory to store checkpointed models')\n",
+    "...\n",
+    "args = parser.parse_args()\n",
+    "\n",
+    "```\n",
+    "When SageMaker training finishes, it deletes all data generated inside the container with exception of the directory _/opt/ml/model_. To ensure that model data is not lost during training, training scripts are invoked in SageMaker with an additional argument **--model_dir**, that needs to be handle by the training script. We need to replace the argument **--save_dir** with the required argument **--model_dir**: "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# this command will replace data_dir by model_dir in the training script\n",
+    "!sed -i 's/save_dir/model_dir/g' char-rnn-tensorflow/train.py"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now, the training script can executed as follow in the container:\n",
+    "\n",
+    "> ```bash\n",
+    "python train.py --num-epochs 1 --data_dir /opt/ml/input/data/training --model_dir /opt/ml/model\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Test locally using SageMaker Python SDK TensorFlow Estimator"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "You can use the SageMaker Python SDK [TensorFlow](https://github.com/aws/sagemaker-python-sdk/blob/master/src/sagemaker/tensorflow/README.rst#training-with-tensorflow) estimator to easily train locally and in SageMaker. To train locally, you set the instance type to [local](https://github.com/aws/sagemaker-python-sdk#local-mode) as follow:"
```

Comment:
I'd change this paragraph from "you can do x" to "here we do x" - does that make sense?